test/functional/Java8andUp/build.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 
 <!--
-  Copyright (c) 2016, 2019 IBM Corp. and others
+  Copyright (c) 2016, 2020 IBM Corp. and others
```

Comment:
Since the only change here is a copyright update, this change can be removed.

Reply:
Agreed and removed the copyright change in this build.